### ISSUE TITLE
Consider using batch-MCTS

### ISSUE THREAD

**Issue by asgerius** [asgerius/rl-rubiks#73]

**Comment by sorenmulli** [comment]
No too important if max_states is used though?

**Comment by asgerius** [comment]
Not really, but it will take forever to evaluate it if given as many states as BWAS needs. Also, batch-MCTS is a slightly different algorithm, which is arguably more comparable to BWAS.
